projects/instant-ui-library/src/lib/loading-spinner/loading-spinner/loading-spinner.component.scss

```diff
@@ -0,0 +1,62 @@
+@import "../../styles/global.styles";
+@import "../../styles/variables/constants";
+
+@mixin setSpinnerSize($spinner-size) {
+  width: $spinner-size;
+  height: $spinner-size;
+}
+
+@mixin setSpinnerColor($spinner-color) {
+  border-top: 3px solid $spinner-color;
+}
+.spinner {
+  border: 3px solid $color--gray1;
+  border-radius: 50%;
+  animation: spin 2s linear infinite;
+
+  &.spinner-small {
```

Comment:
Are you using the conventions of double dash like the other areas? So for example `spinner--small`? Would choose a convention and stick with it.